apps/dashboard/app/helpers/support_ticket_helper.rb

```diff
@@ -2,6 +2,10 @@
 
 # helper for the support ticket pages, email, and request tracker content.
 module SupportTicketHelper
+  def support_ticket_description_text
+    ::Configuration.support_ticket_config[:description]&.html_safe
```

Comment:
I know this has been pending for a bit, I'm just wondering about this part.  This is the default description that'd show up as the placeholder in the text field right?

I'm wondering if we need to support HTML. I wouldn't say that it's risky, but it does carry some risk. In any case, do we _need_ html support is what I'm mulling over.

Reply:
If you're ambivalent then yes, I'd say let's err on the side of caution and remove it.

Reply:
Fixed